persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -52,46 +60,67 @@ public class Init implements BLauncherCmd {
     private final PrintStream outStream = System.out;
     private final String configPath = PersistToolsConstants.CONFIG_SCRIPT_FILE;
 
-    private String name = "";
+    private String configName = "";
+    private String persistName = "";
     public String sourcePath = "";
     private static final String COMMAND_IDENTIFIER = "persist-init";
 
+
     Project balProject;
 
     @CommandLine.Option(names = {"-h", "--help"}, hidden = true)
     private boolean helpFlag;
 
     @Override
     public void execute() {
+        Path persistTomlPath = Paths.get(this.sourcePath, KEYWORD_PERSIST , PERSISTTOML);
```

Comment:
```suggestion
        Path persistTomlPath = Paths.get(this.sourcePath, KEYWORD_PERSIST, PERSISTTOML);
```
Dont we have checkstyles enabled? This should have been caught

Reply:
This doesn't get detected by the checkstyle, it only gets triggered if we missed an space after the comma

Reply:
But better correct it

Reply:
updated in [8415ac4](https://github.com/ballerina-platform/persist-tools/pull/52/commits/8415ac40b0d7895fed58315bbc7624580d898835)